src/System.Net.Http.WinHttpHandler/tests/UnitTests/FakeInterop.cs

```diff
@@ -388,6 +388,15 @@ private static bool CopyToBufferOrFailIfInsufficientBufferLength(string value, I
             return true;
         }
 
+        public static bool WinHttpQueryOption(
+            SafeWinHttpHandle handle,
+            uint option,
+            ref uint buffer,
+            ref uint bufferSize)
+        {
+            return true;
```

Comment:
I meant, for example, that the code at https://github.com/dotnet/corefx/pull/10948/files/3683316544a2b59269d66ab6f4ac4a84d3a8f186#diff-933c557c2433cae9802bf40cf9b4ffbfR364 is initializing the buffer argument to 0 and then after the call is checking to see if it contains a particular value.  This mock version isn't changing the value of that ref arg.